src/main/java/frc/robot/RobotContainer.java

```diff
@@ -49,22 +51,23 @@
 
 
   private final Joystick driverJoystick = new Joystick(Constants.portNumber);
-  private JoystickButton[] mButtons = new JoystickButton[11];
+  private JoystickButton[] buttons = new JoystickButton[11];
 
   // subsystems
   private final DriveBaseSubsystem driveBaseSubsystem = new DriveBaseSubsystem(driverJoystick);
   private final CDSSubsystem CDSSubsystem = new CDSSubsystem();
-  private final IntakeSubsystem IntakeSubsystem = new IntakeSubsystem(); 
-  private final ShooterSubsystem ShooterSubsystem = new ShooterSubsystem();
+  private final IntakeSubsystem intakeSubsystem = new IntakeSubsystem(); 
```

Comment:
Rename to have m_intake.... same for shooterSubsystem

Reply:
See reply below about prefixes, but I would avoid making the variable name the same as the class name. At least make it lowerCamelCase to match the style guide